sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestFramework/DiagnosticVerifier.cs

```diff
@@ -34,50 +38,59 @@
 
 namespace SonarAnalyzer.UnitTest.TestFramework
 {
+
+    class ErrorInUnitTest : Exception
+    {
+#pragma warning disable S3963 // "static" fields should be initialized inline
+        static ErrorInUnitTest()
+        {
+            var msCoreResourceManager = new ResourceManager("mscorlib", typeof(Object).Assembly);
+            at = msCoreResourceManager.GetString("Word_At");
+            inFileLineNum = msCoreResourceManager.GetString("StackTrace_InFileLineNumber");
```

Comment:
Isn't this a bit over-engineered? Two constants with a comment + test would have been enough...

Reply:
It cannot be constant, since depending on the language of installation of visual studio, those strings will change:
- The default exception stack will be in the user language
- The mechanism that parses the exception stack expects to see strings in the user language. If I generate an English string on my machine, it will not work.